convene-web/config/routes.rb

```diff
@@ -30,7 +32,6 @@
   resources :guides, only: %i[index show]
 
   constraints BrandedDomain.new(Space) do
-    root 'spaces#show'
```

Comment:
I assume if I got to `meet.zinc.coop`, this will still works by showing the space with rooms?

Reply:
Yes.